src/Datadog.Trace.ClrProfiler.Managed/Integrations/HttpClientIntegration.cs

```diff
@@ -0,0 +1,768 @@
+        /// <returns>Returns the value returned by the inner method call.</returns>
+        [InterceptMethod(
+        TargetAssembly = SystemNetHttp,
+        TargetType = HttpClientTarget,
+        TargetSignatureTypes = new[] { ClrNames.StreamTask, ClrNames.Uri },
+        TargetMinimumVersion = Major4,
+        TargetMaximumVersion = Major4)]
+        public static object GetStreamAsync(
+            object handler,
+            object uri,
+            int opCode,
+            int mdToken,
+            long moduleVersionPtr)
+        {
+            if (handler == null)
+            {
+                throw new ArgumentNullException(nameof(handler));
+            }
+
+            const string methodName = nameof(GetStreamAsync);
+
+            var callOpCode = (OpCodeValue)opCode;
+            var httpClient = handler.GetInstrumentedType(HttpClientTarget);
+
+            Func<HttpClient, Uri, Task<Stream>> instrumentedMethod = null;
+
+            try
+            {
+                instrumentedMethod =
+                    MethodBuilder<Func<HttpClient, Uri, Task<Stream>>>
+                       .Start(moduleVersionPtr, mdToken, opCode, methodName)
+                       .WithConcreteType(httpClient)
+                       .WithParameters(uri)
+                       .WithNamespaceAndNameFilters(ClrNames.GenericTask, ClrNames.Uri)
+                       .Build();
+            }
+            catch (Exception ex)
+            {
+                Log.ErrorRetrievingMethod(
+                    exception: ex,
+                    moduleVersionPointer: moduleVersionPtr,
+                    mdToken: mdToken,
+                    opCode: opCode,
+                    instrumentedType: HttpClientTarget,
+                    methodName: methodName,
+                    instanceType: handler.GetType().AssemblyQualifiedName);
+                throw;
+            }
+
+            return GetStreamAsyncInternal(
+                instrumentedMethod,
+                reportedType: callOpCode == OpCodeValue.Call ? httpClient : handler.GetType(),
+                (HttpClient)handler,
+                (Uri)uri);
+        }
+
+        private static async Task<Stream> GetStreamAsyncInternal(
+            Func<HttpClient, Uri, Task<Stream>> getStreamAsync,
+            Type reportedType,
+            HttpClient handler,
+            Uri uri)
+        {
+            using (var scope = ScopeFactory.CreateOutboundHttpScope(Tracer.Instance, "GET", uri, IntegrationName))
+            {
+                try
+                {
+                    Stream response = await getStreamAsync(handler, uri).ConfigureAwait(false);
+                    return response;
+                }
+                catch (Exception ex)
+                {
+                    scope?.Span.SetException(ex);
+                    throw;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Instrumentation wrapper for HttpClient.GetAsync.
```

Comment:
```suggestion
        /// Instrumentation wrapper for HttpClient.GetByteArrayAsync.
```

Reply:
HttpClientIntegration.cs has been removed from the project, deleted completely. I'll verify.

Reply:
No longer using HttpClientIntegration.cs